src/itemdb.d

```diff
@@ -117,6 +118,7 @@ final class ItemDatabase
 				remoteDriveId    TEXT,
 				remoteId         TEXT,
 				deltaLink        TEXT,
+				syncStatus       TEXT,
```

Comment:
Is it easier to keep the status as text? I mean, that is not the really optimal storage type for a Boolean, and we only save Y or N, right?

Reply:
Is it easier? Probably the same as boolean storage wise - as a bool would be stored as 0x00 / 0xFF (or whatever sqlite uses) vs 0x59 / 0x4E .. so same amount of memory space IMHO .. happy to look at changing that to bool tho if you think thats what should be used.

Reply:
Not that I really care, I'm just used to optimize db tables due to lots of big data stuff I am doing ;-) I'm fine with it if you are fine with it ;-)